apis/atmos/src/main/java/org/jclouds/atmos/AtmosApiMetadata.java

```diff
@@ -57,6 +58,7 @@ public static Properties defaultProperties() {
       Properties properties = BaseHttpApiMetadata.defaultProperties();
       properties.setProperty(PROPERTY_REGIONS, "DEFAULT");
       properties.setProperty(PROPERTY_USER_METADATA_PREFIX, "X-Object-Meta-");
+      properties.setProperty(Constants.PROPERTY_IDEMPOTENT_METHODS, "DELETE,GET,HEAD,OPTIONS,POST,PUT");
```

Comment:
[nit] Use a static import like in the other properties (consider this also int he other classes)